target_chains/ethereum/contracts/forge-test/Entropy.t.sol

```diff
@@ -696,4 +696,30 @@ contract EntropyTest is Test, EntropyTestUtils {
         assertEq(providerInfo1.uri, provider1Uri);
         assertEq(providerInfo1.commitmentMetadata, provider1CommitmentMetadata);
     }
+
+    function testSetProviderFee() public {
+        vm.prank(provider1);
+        random.setProviderFee(1);
+
+        assertEq(random.getProviderInfo(provider1).feeInWei, 1);
```

Comment:
can you check that the fee isn't 1 before this test? 

I don't like tests that can be broken accidentally by people changing outside of the test method.